api/src/test/java/org/openmrs/api/impl/UserServiceImplTest.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.impl;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.User;
+import org.openmrs.api.PasswordException;
+import org.openmrs.api.UserService;
+import org.openmrs.api.context.Context;
+import org.openmrs.test.BaseContextSensitiveTest;
+
+public class UserServiceImplTest extends BaseContextSensitiveTest {
+
+	UserService userService;
+	User user;
+	
+	@Before
+	public void before() {
+		userService = Context.getUserService();
+		user = new User();
+		user.setUsername("john");
+		user.setSystemId("1");
+	}
+	
+	@Test(expected = PasswordException.class)
```

Comment:
Use junit's @ExpectedException instead of this because it allows to check the actual error message since there is several sub classes of this with different error messages